test/runnable/structlit.d

```diff
@@ -323,9 +325,9 @@ int waz14(S)(ref S s) { return 2; }
 
 void test14()
 {
+    foo14(S14a(0));
+    foo14(S14b(0));
     // can not bind rvalue-sl with ref
-    static assert(!__traits(compiles, foo14(S14a(0))));
-    static assert(!__traits(compiles, foo14(S14b(0))));
     static assert(!__traits(compiles, hoo14(S14a(0))));
```

Comment:
Why doesn't this compile? It seem the only difference with the `foo14` test is that it's a template.

Reply:
Because it seems that none of the template checking code calls `functionParameters`. I agree that it should work, I'm just not sure where the equivalent code for templates is.